CharlieBackend.Api/Validators/AccountDTOValidators/AuthenticationDtoValidator.cs

```diff
@@ -15,8 +13,10 @@ public AuthenticationDtoValidator()
                 .EmailAddress().WithMessage("Incorrect email")
                 .MaximumLength(50).WithMessage("Email cannot be greateh than {MaxLength} symbols");
             RuleFor(x => x.Password)
-                .NotEmpty().WithMessage("Password cannot be empty")
-                .MaximumLength(65).WithMessage("Password cannot be greater than {MaxLength} symbols");
+                .NotEmpty()
+                .MinimumLength(8)
+                .MaximumLength(30)
+                .Must(PasswordHelper.PasswordValidation).WithMessage("Password must have at least eight characters, at least one uppercase letter, one lowercase letter and one number");
```

Comment:
> "Password must have at least eight characters, at least one uppercase letter, one lowercase letter and one number"

This error string occurs in every validator with a password. Perhaps we should introduce string constants for common error messages

Reply:
Done in https://github.com/ita-social-projects/WhatBackend/pull/435/commits/0c41ce8c05fe9bcf77febbc04a6fae5a668c4d6f